tests/unittest_brain.py

```diff
@@ -1218,18 +1218,15 @@ def test_typing_alias_type(self):
 
         def check_metaclass(node: nodes.ClassDef):
             meta = node.metaclass()
-            assert (
-                isinstance(meta, nodes.ClassDef)
-                and meta.name == "ABCMeta_typing"
-                and "ABCMeta" == meta.basenames[0]
-                and meta.locals.get("__getitem__") is not None
-            )
+            assert isinstance(meta, nodes.ClassDef)
```

Comment:
👍

Reply:
I really hope all the tests pass now. Was a bit more difficult to fix than I expected.

Reply:
@Pierre-Sassoulas Can you reproduce the Travis issue. I unfortunately can't 😞

Reply:
Hmm it's strange, on 239106101bd3fddee25220ecb5883d62c7738129 I have an error but not the same than for Travis
```
____ ExceptionModelTest.test_oserror __

self = <tests.unittest_object_model.ExceptionModelTest testMethod=test_oserror>

    def test_oserror(self):
        ast_nodes = builder.extract_node(
            """
        try:
            raise OSError("a")
        except OSError as err:
           err.filename #@
           err.filename2 #@
           err.errno #@
        """
        )
        expected_values = ["", "", 0]
        for node, value in zip(ast_nodes, expected_values):
            inferred = next(node.infer())
>           assert isinstance(inferred, astroid.Const)
E           AssertionError: assert False
E            +  where False = isinstance(Uninferable, <class 'astroid.node_classes.Const'>)
E            +    where <class 'astroid.node_classes.Const'> = astroid.Const
```

Reply:
Hmm. Should I just change the required version to `3.8`?
Since it works with `AppVeyor` [Link](https://ci.appveyor.com/project/PCManticore/astroid/builds/37990402/job/9l8l6x0o8ybx8odj), I would assume that this is a Travis issue.